.travis.yml

```diff
@@ -47,27 +47,27 @@ jobs:
     - stage: coverage
       env:
         - TESTING_PROFILE=unit-tests
-      script: mvn --batch-mode clean install jacoco:report jacoco:report-aggregate coveralls:report -P$TESTING_PROFILE,coverage
+      script: mvn --batch-mode clean install org.jacoco:jacoco-maven-plugin:report org.jacoco:jacoco-maven-plugin:report-aggregate coveralls:report -P$TESTING_PROFILE,coverage
```

Comment:
Why did the prefixes become necessary?

Reply:
The build failed because maven could not locate the jacoco plugin via the command line. I don't know why this suddenly became an issue but the only way I could find to solve it was to completely specify the artifact and group ids. See this post http://tdongsi.github.io/blog/2017/09/23/jacoco-in-maven-project/